File: crud_web_auto_reencauches_1/src/main/java/com/mycompany/crud_web_auto_reencauches_1/Cliente/Entity/Cliente.java
package com.mycompany.crud_web_auto_reencauches_1.Cliente.Entity;

import javax.persistence.*;

@Entity
public class Cliente {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private int id;

    private String nombre;

    private String apellido;

    private long cedula;

    private long telefono;

    private String email;

    public Cliente() {
    }

    public Cliente(String nombre, String apellido, long cedula, long telefono, String email) {
        this.nombre = nombre;
        this.apellido = apellido;
        this.cedula = cedula;
        this.telefono = telefono;
        this.email = email;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public String getApellido() {
        return apellido;
    }

    public void setApellido(String apellido) {
        this.apellido = apellido;
    }

    public long getCedula() {
        return cedula;
    }

    public void setCedula(long cedula) {
        this.cedula = cedula;
    }

    public long getTelefono() {
        return telefono;
    }

    public void setTelefono(long telefono) {
        this.telefono = telefono;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

}

File: Proyecto-de-Rencauche-de-Llantas-con-Angular/src/app/Modelo/Cliente.ts
export class Cliente {
    id?: number;
    nombre: string;
    apellido: string;
    cedula: number;
    telefono: number;
    email: string;
    rol: string;
    usuario: string;
    password: string;
    repeat_password: string;

    constructor(nombre: string, apellido: string,
        cedula: number, telefono: number, 
        email: string, usuario: string) {
            this.nombre = nombre;
            this.apellido = apellido;
            this.cedula = cedula;
            this.telefono = telefono;
            this.email = email;
            this.usuario = usuario;
    }
}